recipes/simdjson/0.4.6/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.9)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
```suggestion
```

CCI bot will block it.

I used to enable it when building on my machine, but for regular builds, it only increases the build log and people don't read.

Reply:
Done. Fixed in a follow-up commit.